src/test/java/org/cy3sbml/ols/OLSAccessTest.java

```diff
@@ -22,12 +29,15 @@ public void getTerm() throws Exception {
     }
 
     @Test
-    public void getTermCompact() throws Exception {
-        String resourceURI = "http://identifiers.org/GO:0042752";
-        String identifier = RegistryUtilities.getIdentifierFromURI(resourceURI);
-
-        Term term = OLSAccess.getTerm(identifier);
-        assertNotNull(term);
+    public void getTermCompact() {
+        List<String> resourceURIs = new ArrayList<>();
+        resourceURIs.add("http://identifiers.org/GO:0042752");
```

Comment:
This should be a parametrized test which you parametrize with different URLs. 
Tests should be granular and fail on small units. I.e. you never test the individual URLs, but the test fails if any of the URLS fails.